### PR TITLE
[E2E test] Drawer open/close

### LINKED ISSUE #746: [E2E test] Drawer open/close
# 🧪E2E test cases

The scope of these tests is to ensure that the drawer opens and closes successfully.

Screen Shot 2020-07-21 at 4 28 31 PM

# Test case 1

## 😎 Role

All test should run as admin.

## 🧫 Purpose of the test

Click on the hamburger menu (the 3 lines on the top of the drawer) icon should open the drawer and the user should see an extended view of the icons with descriptions.

## 🐾 Steps

1. Login with admin role
2. Click the hamburger menu icon on the drawer

## Expected result

Drawer opens

# Test case 2

## 😎 Role

All test should run as admin.

## 🧫 Purpose of the test

Click on the X icon should close the extended view of the drawer.

## 🐾 Steps

1. Login with admin role
2. Click the hamburger menu icon on the drawer
3. Click the X icon to close the extended view

## Expected result

Drawer Closes

### PR DESCRIPTION
**Pull request by daniel-shuy**
Closes #746 

Currently I'm hardcoding the expanded/collapsed widths. Should I import the constants from `globalStyles.ts` instead? eg.
```ts
import { globals } from '../../../src/globalStyles'

// ...

describe('Drawer', () => {
  it('clicking on the hamburger menu icon should open the drawer', () => {
    // ...

    cy.get('[data-test="drawer"]').should('have.css', 'width', `${globals.common.drawerWidthExpanded}px`)
  })

  it('clicking on the X icon should close the extended view of the drawer', () => {
    // ...

    cy.get('[data-test="drawer"]').should('have.css', 'width', `${globals.common.drawerWidthCollapsed}px`)
  })
})
```

### CODE REVIEW

**Comment by herflis** [comment]
@daniel-shuy yes, that would be great if you could import globalStyles' constants, thanks 👍

File: apps/sensenet/cypress/integration/drawer/drawer.spec.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { pathWithQueryParams } from '../../../src/services/query-string-builder'
+
+describe('Drawer', () => {
+  it('clicking on the hamburger menu icon should open the drawer', () => {
+    cy.login()
```
Comment (herflis) [inline apps/sensenet/cypress/integration/drawer/drawer.spec.ts:5]: the two login calls could go out into a `beforeEach` :)

File: apps/sensenet/cypress/integration/drawer/drawer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+
+    cy.get('[data-test="drawer-expandcollapse-button"]').click()
+
+    cy.get('[data-test="drawer"]').should('have.css', 'width', '250px')
+  })
+
+  it('clicking on the X icon should close the extended view of the drawer', () => {
+    cy.login()
```
Comment (herflis) [inline apps/sensenet/cypress/integration/drawer/drawer.spec.ts:14]: the two login calls could go out into a `beforeEach` :)

**Comment by daniel-shuy** [comment]
How can I get the build to pass? Even after authorizing Netlify CLI, it fails with HTTP 401

**Comment by herflis** [comment]
The builds probably won't run because it's a forked repository, I don't think you have anything to do with it now 🙂 So thank you for taking the time to help us! We greatly appreciate your contribution, you will get the extra sensenet-Hacktoberfest swag 🧦. We will contact you via email after the Hacktoberfest event. Hope you are having a great Hacktoberfest! 🎉 🙂

**Pull request merged**